src/plone/app/multilingualindexes/__init__.py

```diff
@@ -1,10 +1,21 @@
 # -*- coding: utf-8 -*-
 from zope.i18nmessageid import MessageFactory
+from plone.app.querystring import queryparser
 
 from . import patches  # noqa
 
 _ = MessageFactory("plone.app.multilingualindexes")
 
+try:
+    # Adding 'tgpath' in plone.app.querystring for
+    # multipath functionality
+    queryparser.PATH_INDICES |= {'tgpath'}
+except AttributeError:
+    raise ImportError(
+        "plone.app.querystring doesn't support "
+        "multipath functionality."
+    )
+
```

Comment:
try expect is superfluous since you told in setup.py to use the version with this feature